### PR TITLE
Fix homepage banner and closing soon message dates

### PR DESCRIPTION
**Pull request by tahsinaislam**
## Link to pivotal/JIRA issue
The banner on the homepage message and also the closing soon text message and email have hard coded dates that should be coming from the config dates that have been set up. [Jira ticket](https://codeforamerica.atlassian.net/browse/GYR1-4)

## What was done?
The dates in the banner were taken from the config file instead of being hard coded, the date for the April 1st document deadline was added to the config file and the closing soon messages now all have the dates coming in from the config instead of being hard coded 

## How to test?
Test banner when open for intake in order to make sure the dates are correct still and also test closing soon messages

### CODE REVIEW

**Review by tofarr — APPROVED** [review]
Suggesting changing `first_end_of_docs` to `doc_submission_deadline`

**Comment by tofarr** [comment]
Using the constant makes sense 🍰

**Comment by embarnard** [comment]
if you need to fix for future use remove "new" from "still_needs_help_service.rb" in 
```
ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
      client: client,
      locale: client.intake.locale,
      message: AutomatedMessage::ClosingSoon.new
    )
```

File: app/services/still_needs_help_service.rb [hunk 1]
```diff
@@ -14,7 +14,9 @@ def self.send_still_need_help_notification(client)
     ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
       client: client,
       locale: client.intake.locale,
-      message: AutomatedMessage::ClosingSoon.new
+      message: AutomatedMessage::ClosingSoon.new,
```
Comment (embarnard) [inline app/services/still_needs_help_service.rb:17]: remove "new" from this line

**Pull request merged**